ci-operator/step-registry/azure/provision/service-principal/minimal-permission/azure-provision-service-principal-minimal-permission-ref.yaml

```diff
@@ -27,6 +27,10 @@ ref:
     default: "false"
     documentation: |-
       Assign additional permissions for SP to install cluster configured disk encryption set
+  - name: ENABLE_MIN_PERMISSION_FOR_STS
```

Comment:
Will this flag be applied to all supported platforms?

Reply:
Does it require such ENV on aws platform? If not, maybe it can be renamed to reflect specific platform.
But in this ref yaml file, other similar envs `ENABLE_MIN_PERMISSION_FOR_MARKETPLACE`, `ENABLE_MIN_PERMISSION_FOR_DES` are defined, so naming this STS env with similar rule.

Reply:
No control flags were used on AWS so far, I'm thinking if adopt `ENABLE_MIN_PERMISSION_FOR_STS` for it.

Reply:
yes, that might be a good idea for AWS too